src/Annotations/AppKernel.php

```diff
@@ -224,7 +224,10 @@ public function __construct(string $environment = 'dev')
 
             return new Client([
                 'base_uri' => $this->app['hypothesis']['api_url'],
+                'connect_timeout' => 0.5,
+                'decode_content' => 'gzip',
```

Comment:
Probably worth doing as this is external.

Reply:
Just confirmed that the current hypothesis-dummy supports this for testing (happens at the cURL level, which the local tests don't hit), though https://github.com/elifesciences/hypothesis-dummy/pull/3 will turn it off (removes Nginx; could add it to the PHP code?).

Reply:
Should degrade automatically if the response is not gzipped as there's no `Content-Encoding` header? Which would be a difference between `end2end` and `prod` but not critical?

Reply:
Yep, cURL handles it all.

Reply:
As it stands, this should not explode with any version of `hypothesis-dummy` then